src/main/java/com/mongodb/jdbc/BsonTypeInfo.java

```diff
@@ -0,0 +1,183 @@
+package com.mongodb.jdbc;
+
+import java.sql.SQLException;
+
+public class BsonTypeInfo {
+    public static final String BSON_BINDATA_TYPE_NAME = "binData";
```

Comment:
```suggestion
    public static final String BINDATA_TYPE_NAME = "binData";
```
[ultra-nit] Totally up to you (or anyone else who has a strong opinion): we could remove the BSON prefix from all of these since they'll be accessed using the class name (`BsonTypeInfo.XYZ_TYPE_NAME`).